docs/source/create_channel/create_channel_participation.md

```diff
@@ -0,0 +1,550 @@
+                          Rule: "OR('SampleOrg.member')"
+        Application:
+            <<: *ApplicationDefaults
+            Organizations:
+                - <<: *SampleOrg
+                  Policies:
+                      <<: *SampleOrgPolicies
+                      Admins:
+                          Type: Signature
+                          Rule: "OR('SampleOrg.member')"
+
+```
+
+**Note:** For simplicity, this snippet assumes that the peers and orderers belong to the same organization `SampleOrg`. You can refer to the [sample config](https://github.com/hyperledger/fabric/blob/{BRANCH}/sampleconfig/configtx.yaml) for the full example. For a production deployment however, it is recommended that the peer and ordering nodes belong to separate organizations.
+
+An unlimited number of profiles can be listed in the `Profiles` section according to the needs of your use cases. However, each must have a distinctive name (this will not become the name of the channel itself, as that is specified by a flag given when the channel is created). In the next step, you specify which profile to use when you generate the channel genesis block.
+
+### Generate the genesis block
+
+After you have completed editing the `configtx.yaml`, you can start to create a new channel for the peer organizations. Every channel configuration starts with a genesis block. Because we previously set the environment variables for the `configtxgen` tool, you can run the following command to build the genesis block for `channel1` using the `SampleAppChannelEtcdRaft` profile:
+
+```
+configtxgen -profile SampleAppChannelEtcdRaft -outputBlock ./channel-artifacts/channel1.tx -channelID channel1
+```
+
+Where:
+- `-profile` -  Name of the profile in `configtx.yaml` that will be used to create the channel.
+- `-outputBlock` - Where to store the generated configuration block file.
+- `-channelID` - Name of the channel being created. Channel names must be all lower case, less than 250 characters long and match the regular expression ``[a-z][a-z0-9.-]*``. The command uses the `-profile` flag to reference the `SampleAppChannelEtcdRaft:` profile from `configtx.yaml`.
+
+When the command is successful, you will see logs of `configtxgen` loading the `configtx.yaml` file and printing a channel creation transaction:
+```
+INFO 001 Loading configuration
+INFO 002 orderer type: etcdraft
+INFO 003 Loaded configuration: /fabric/config/configtx.yaml
+INFO 004 Generating genesis block
+INFO 005 Writing genesis block
+
+```
+
+## Step two: Use the `osnadmin` CLI to start the channel
+
+Now that the genesis block has been created, the first ordering node that receives the `osnadmin channel join` command command effectively "activates" the channel, though the channel is not fully operational until a quorum of consenters is established (if your profile listed three consenters, at least one more node, for a total of two, would have to join using the `osnadmin channel join` command).
```

Comment:
I'm really not sure what "activates" is supposed to mean.  If you subsequently remove the channel from the first orderer, is it still activated? From a peer perspective, how does is an "activated" channel different from one that is not?

I guess the question really is "If a channel genesis block is created, but nobody joins it, does the channel exist?"  I would argue that it does.

Reply:
I added this --- I was trying to avoid the philosophical question about when a channel is really "created". Is it when the genesis block is created or when nodes are joined to it or is it when enough nodes are joined to it. This is basically saying that, yes, a genesis block has been created, but the channel isn't really "active" until nodes are joined to it. Sort of splitting the difference.

Anyway, I think if we come to broader agreement about what terms to use in these situations that will be clear up what terminology should be used here and elsewhere.

Reply:
Welcome better suggestions there.